### PR TITLE
[typescript] Add examples

### PR DESCRIPTION
**Pull request by amayvs**

### CODE REVIEW

File: typescript/src/app.ts [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  const account: Account = {
+    address: process.env.ACCOUNT_ADDRESS || "",
+    publicKey: process.env.PUBLIC_KEY || "",
+    privateKey: process.env.PRIVATE_KEY || "",
+    ethereumAccount: process.env.ETHEREUM_ACCOUNT || "",
+  };
+
+  try {
+    await onboardUser(config, account);
+    account.jwtToken = await authenticate(config, account);
```
Comment (SergeyVolynkin) [inline typescript/src/app.ts:32]: We may also provide an example of refreshing the auth token every 3-4 mins as the token is short-lived;
But probably for the simplicity of the example, it is better to keep as-is;

File: typescript/src/app.ts [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+(async () => {
+  // TODO: Get from /system/config
+  const apiBaseUrl = "https://api.testnet.paradex.trade/v1";
+  const chainId = shortString.encodeShortString("PRIVATE_SN_POTC_SEPOLIA");
+  const config: SystemConfig = {
+    apiBaseUrl,
+    starknet: { chainId },
+  };
+
+  // TODO: Add key derivation
```
Comment (SergeyVolynkin) [inline typescript/src/app.ts:22]: Do we want to follow up on these TODOs before merging? (In my understanding, we can follow up later 👍)

**Review by SergeyVolynkin — APPROVED** [review]
Thank you @amayvs! Good to 🚢

File: typescript/src/app.ts (hunk 2 above)
Comment (amayvs) [reply typescript/src/app.ts:22]: Will be raising follow-up PRs for these

File: typescript/src/app.ts (hunk 1 above)
Comment (amayvs) [reply typescript/src/app.ts:32]: I'll add a follow-up task

**Pull request merged**